programs/sp1-programs/update-client/src/lib.rs

```diff
@@ -34,7 +34,13 @@ pub fn update_client(
         clock_drift: Duration::default(),
     };
 
-    let ctx = types::validation::ClientValidationCtx::new(time, &trusted_consensus_state);
+    let mut ctx = types::validation::ClientValidationCtx::new(time);
```

Comment:
We could use a builder pattern to avoid this `mut`, but its probably not worth it.